cms/static/js/factories/login.js

```diff
@@ -22,6 +22,8 @@ define(['jquery.cookie', 'utility'], function() {
 
         $('form#login_form').submit(function(event) {
             event.preventDefault();
+            $('#submit').addClass('disabled');
```

Comment:
you should use our existing helper function: https://github.com/edx/edx-platform/blob/master/common/static/common/js/components/utils/view_utils.js#L111-L123